### ISSUE TITLE
Higher heterozygosity

### ISSUE THREAD

**Issue by shehongbing** [schatzlab/genomescope#109]
hi, there, I performed genome survey  using HiFi reads. And my command as follows:

kmc -k27 -m24 -cs10000 pigweed.HiFi.fq.gz pigweed ./
kmc_tools transform pigweed histogram pigweed.histo -cx10000

The results as follows:
http://qb.cshl.edu/genomescope/genomescope2.0/analysis.php?code=ffiaSWwLPRS0FLXEUp1D

I was puzzled about the high heterozygosity (7.34%), but almost one peak is found. Would you mind give me some suggestions.

thx
hongbing

**Comment by mschatz** [comment]
I agree it is not a great fit. I think it is getting confused by the
relatively high rate of error. Have you used DeepConsensus on the reads?
That often cleans up the data nicely

Good luck!
Mike
